tools/docgenerator.js

```diff
@@ -560,17 +560,18 @@ function collectDocs (functionNames, inputPath) {
     if (path.indexOf('docs') === -1 && functionIndex !== -1) {
       if (path.indexOf('expression') !== -1) {
         category = 'expression'
-      } else if (/^.\/lib\/type\/[a-zA-Z0-9_]*\/function/.test(fullPath)) {
+      } else if (/\/lib\/cjs\/type\/[a-zA-Z0-9_]*\/function/.test(fullPath)) {
+        // for type/bignumber/function/bignumber.js, type/fraction/function/fraction.js, etc
```

Comment:
Hmm, the test `path.indexOf('type') !== -1` works just as well for me here and is much easier on the eyes...